packages/peregrine/lib/talons/RootComponents/Category/useCategoryContent.js

```diff
@@ -62,16 +63,71 @@ export const useCategoryContent = props => {
     const [, { dispatch }] = useEventingContext();
 
     useEffect(() => {
+        const fashionColor = [];
+        const fashionMaterial = [];
+        const fashionSize = [];
+        const fashionStyle = [];
+        const hasVideo = [];
+
+        //{from: "40" to: "59"}
+
+        if (filterOptions) {
+            for (const [group, items] of filterOptions) {
```

Comment:
All these product attributes are custom and not present in all stores. Hard coding them this way is the wrong approach in my opinion.